libs/s25main/ingameWindows/iwMapGenerator.cpp

```diff
@@ -30,19 +30,20 @@ using namespace rttr::mapGenerator;
 
 enum
 {
-    CTRL_LAST_ID = 6, // last UI control ID used before enum controls
+    CTRL_LAST_ID = 7, // last UI control ID used before enum controls
```

Comment:
Just a hint: To avoid such changes, simply give all IDs an enum value, even the texts.